### ISSUE TITLE
Support elastic tabstops

### ISSUE THREAD

**Issue by erbridge** [limetext/lime#479]
It would be good for us to be able to support elastic tabstops as an alternative to space- or tab-indentation. See [here](http://nickgravgaard.com/elastictabstops/) and [here](http://nickgravgaard.com/elastictabstops/news/elastic-tabstops-lite/).

**Comment by quarnster** [comment]
Good idea. Also looks like there is an implementation in the standard [text/tabwriter](http://golang.org/pkg/text/tabwriter/) package
